cypress/support/commands.ts

```diff
@@ -32,6 +32,27 @@
 //   }
 // }
 
-// Cypress.Commands.add("clearDb", () => {
-//   cy.exec("poetry run python manage.py flush --no-input");
-// });
+type UserData = {
+  sessionid: string;
+  csrftoken: string;
+};
+
+const sessions: Record<string, UserData> = {
+  webkom: {
+    sessionid: "ln6y4kwaqp3y5mxv1ft19p6or1a018xq",
+    csrftoken: "vKaIGVmBwUPuNkZyvFVy0Ep2IH4B0mG4", // bNnzylsDzcNAY4YZdJL33vai22T7NX0AVqbbHXtJVe7MNt4FeJnV5ZQpu5YbinZU
```

Comment:
What's up with the commented token?

Reply:
Good catch, that was just something I saved while I was playing around - will remove